src/helpers/MeasureBundleHelpers.ts

```diff
@@ -45,6 +45,19 @@ export function isEpisodeOfCareGroup(measure: fhir4.Measure, group: fhir4.Measur
   }
 }
 
+/**
+ * Measure Observation populations can specify which population in the measure they are drawing observations from (e.g. Numerator/Denominator/etc.)
+ * This is done by referencing the ID of that population in the valueString of the cqfm-criteriaReference extension
+ * This function identifies the ID used in one of those measure observation populations
+ */
```

Comment:
Gonna slightly disagree with this, as IMO using types in JSDocs is unnecessary when using typescript since the function signature itself is basically self documenting.

I'll ad a description in the comment though about the param and return value